packages/sdk/src/logs.ts

```diff
@@ -59,3 +59,26 @@ export function parseLogs(input: unknown): readonly Log[] {
   if (!Array.isArray(input)) throw new Error("Logs must be an array");
   return input.map(parseLog);
 }
+
+/**
+ * Searches in logs for the first event of the given event type and in that event
+ * for the first first attribute with the given attribute key.
+ *
+ * Throws if the attribute was not found.
+ */
+export function findAttribute(
```

Comment:
Yeah, a few lines, but super nice to expose as a function

Reply:
It was not possible to copy it over (which I tried first) since nodejs 12 does not support JavaScript's new optional chaining syntax. This way, it gets processed and prebuilt by TS.